tests/e2e/specs/dashboard/screens.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Dashboard screens test.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { enableFeature, setupSiteKit } from '../../utils';
+
+describe( 'dashboard screens', () => {
+	it( 'ensures that the page title element pattern is correct for screens with private (and non-null) parent_slug values', async () => {
+		await enableFeature( 'userInput' );
+
+		await setupSiteKit();
+
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-user-input' );
+
+		await page.waitForSelector( 'title' );
+
+		await expect( page ).toMatchElement( 'title', {
+			text: 'Site Kit by Google User Input ‹ Google Site Kit Dev — WordPress',
```

Comment:
Nice to have: it would be nice if the test wasn't coupled to the WP core-specific part of the title. I.e. How about we change `text` here to use a `RegExp` that checks that the text starts with our expected title, but doesn't include the WP core part? Not critical, but would help to make the test a little less fragile.

Reply:
Thanks @aaemnnosttv this has been resolved in a recent commit, I've moved to using a simple regex pattern that checks that the first part of the title begins with what is expected, this should suffice as it was the only part missing prior.